downloads/oldreleases.jl

```diff
@@ -71,9 +82,16 @@ open("./oldreleases.md", "w") do io
             """)
     for (version, files) in releases
         n = length(files)
+        release_date = ""
```

Comment:
I think this block is just `get(release_dict, version, "Unknown")`

Reply:
Awesome, switched it over.